app/src/test/java/ch/epfl/sdp/appart/DatabaseUnitTest.java

```diff
@@ -34,4 +36,29 @@ public void updateCardTest() {
         db.updateCard(card);
         assertTrue(true);
     }
+
+    @Test
+    public void getUserTest() {
+        Database db = mock(Database.class);
+        db.getUser("id0");
+        assertTrue(true);
+    }
```

Comment:
Just to let you know, you don't necessarily need Mockito to mock the database. You can actually do as in the ScrollActivityTest and simply uninstall the FirebaseDB.class module and then rebind the MockDatabase.

Reply:
this test won't change the coverage, btw

Reply:
in general `DatabaseUnitTest` is a class that should be removed, I wrongly added it during the first sprint